dspace-spring-rest/src/main/java/org/dspace/app/rest/model/FacetResultsRest.java

```diff
@@ -33,7 +35,15 @@ public SearchFacetEntryRest getFacetEntry() {
         return facetEntry;
     }
 
-    public void setFacetEntry(final SearchFacetEntryRest facetEntry) {
+    public void setFacetEntry(SearchFacetEntryRest facetEntry) {
         this.facetEntry = facetEntry;
     }
+
+    public void setPrefix(String prefix) {
```

Comment:
Needs javadocs to describe what a Prefix is.

Reply:
We'll add comments here similar to https://github.com/DSpace/DSpace/blob/dspace-6.3/dspace-api/src/main/java/org/dspace/discovery/DiscoverFacetField.java#L21